### PR TITLE
Updating version, hash and doc url.

### PR DESCRIPTION
**Pull request by ryanskeith**
Seems straightforward:
Updated version, hash and doc url

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   skip: true  # [py<36]
```
Comment (marcoesters) [inline recipe/meta.yaml]: python 3.6 is not supported anymore: https://github.com/aio-libs/multidict/blob/v6.0.2/setup.py#L63-L67

File: recipe/meta.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ about:
 
   description: |
     Multidicts are useful for working with HTTP headers, URL query args etc.
-  doc_url: https://multidict.readthedocs.io/
```
Comment (marcoesters) [inline recipe/meta.yaml:49]: I would remove `en/stable` where possible so that we don't prevent internationalization.

**Review by varlackc — COMMENTED** [review]
since `pip` is mentioned in the recipe we might want to add `pip chech` in the test section.

https://github.com/AnacondaRecipes/multidict-feedstock/blob/74af8c16abbcc388a60c5da2c6e507561822464b/recipe/meta.yaml#L37

File: recipe/meta.yaml [hunk 3]
```diff
@@ -34,6 +34,7 @@ test:
   requires:
     - pytest
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:35]: Missing `pip` in `test/requires`

**Pull request merged**